packages/caliper-vscode/artifacts/benchmark.json

```diff
@@ -0,0 +1,313 @@
+            "additionalProperties": false,
+            "properties": {
+                "test": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "description": "The settings related to the benchmark workload all reside under the root test attribute, which has some general child attributes, and the important rounds attribute.",
+                    "properties": {
+                        "name": {
+                            "description": "Short name of the benchmark to display in the report.",
+                            "type": "string"
+                        },
+                        "description": {
+                            "description": "Detailed description of the benchmark to display in the report.",
+                            "type": "string"
+                        },
+                        "workers": {
+                            "type": "object",
+                            "description": "Object of worker-related configurations.",
+                            "additionalProperties": false,
+                            "properties": {
+                                "number": {
+                                    "description": "Specifies the number of worker processes to use for executing the workload.",
+                                    "type": "integer"
+                                },
+                                "type": {
+                                    "description": "Currently unused.",
+                                    "type": "null"
+                                }
+                            },
+                            "required": [
+                                "number"
+                            ],
+                            "title": "Workers"
+                        },
+                        "rounds": {
+                            "type": "array",
+                            "description": "Array of objects, each describing the settings of a round.",
+                            "items": {
+                                "type": "object",
+                                "additionalProperties": false,
+                                "properties": {
+                                    "label": {
+                                        "description": "A short name of the rounds, usually corresponding to the types of submitted TXs.",
+                                        "type": "string"
+                                    },
+                                    "txNumber": {
+                                        "description": "The number of TXs Caliper should submit during the round.",
+                                        "type": "integer"
+                                    },
+                                    "rateControl": {
```

Comment:
Depending on the value of `type`, the `opts` object can be different. I'd suggest exploring conditional schemas to handle such cases flexibly: https://json-schema.org/understanding-json-schema/reference/conditionals.html#if-then-else
This will require a newer JSON schema version, but vscode supports even the newest draft, so I'd suggest that every schema should use that newest draft.